External/FEXCore/Source/Interface/Core/LLVMJIT/LLVMCore.cpp

```diff
@@ -85,6 +94,9 @@ class LLVMJITCore final : public CPUBackend {
     llvm::Function *CPUIDFunction;
     llvm::Function *ExitVMFunction;
     llvm::Function *ValuePrinter;
+#ifdef _M_ARM_64
+    llvm::Function *AArch64ReadCycleCounterFunction;
```

Comment:
Doesn't llvm have some intrinsic for this?

Reply:
Yea. It's actually explained in a comment down on line 274.